Snowflake.Data.Tests/SFPutGetTest.cs

```diff
@@ -32,6 +35,7 @@ class SFPutGetTest : SFBaseTest
         [ThreadStatic] private static StageType _stageType;
         [ThreadStatic] private static string _compressionType;
         [ThreadStatic] private static bool _autoCompress;
+        [ThreadStatic] private static List<string> _filesToDelete;
```

Comment:
should you follow convention with `t_` prefix?

Reply:
Done